hbase-procedure/src/main/java/org/apache/hadoop/hbase/procedure2/RemoteProcedureDispatcher.java

```diff
@@ -106,6 +106,10 @@ public boolean start() {
     return true;
   }
 
+  protected void setUncaughtExceptionHandler(UncaughtExceptionHandler eh) {
```

Comment:
Better name it something else? Like setTimeoutExecutorUncaughtExceptionHandler? As we have a method all getUncaugtExceptionHandler in this class, for getting uncaughtExceptionHandler for the thread pool.